src/controllers/pageController.ts

```diff
@@ -0,0 +1,83 @@
+import { PageLink } from '../models';
+import { PageLinkAttributes, PageLinkInstance } from '../models/PageLink';
+
+/**
+ * Creates a page with the given attribues
+ *
+ * @param {PageLinkAttributes} attributes
```

Comment:
Can each JSDoc comment have the following properties:
- A short description!
- [`param`](http://usejsdoc.org/tags-param.html)
- [`async`](http://usejsdoc.org/tags-async.html) if applicable
- [`returns`](http://usejsdoc.org/tags-returns.html)
- [`author`](http://usejsdoc.org/tags-author.html)

@tanzeelak is also working on this for her PR (#27)!